packages/layout-grid/Cell.js

```diff
@@ -0,0 +1,46 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Cell = (props) => {
+  const {align, children, className, columns, desktop, order, phone, tablet, ...otherProps} = props;
+
+  const classes = classnames('mdc-layout-grid__cell', className, {
+    [`mdc-layout-grid__cell--align-${align}`]: !!align,
+    [`mdc-layout-grid__cell--order-${order}`]: !!order,
+    [`mdc-layout-grid__cell--span-${columns}`]: !!columns,
+    [`mdc-layout-grid__cell--span-${desktop}-desktop`]: !!desktop,
+    [`mdc-layout-grid__cell--span-${phone}-phone`]: !!phone,
+    [`mdc-layout-grid__cell--span-${tablet}-tablet`]: !!tablet,
+  });
+
+  return (
+    <div className={classes} {...otherProps}>
+      {children}
+    </div>
+  );
+};
+
+Cell.propTypes = {
+  align: PropTypes.oneOf(['bottom', 'middle', 'top']),
+  children: PropTypes.node,
+  className: PropTypes.string,
+  columns: PropTypes.oneOf([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
+  desktop: PropTypes.oneOf([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
```

Comment:
can we change the name of desktop, phone, and tablet to `desktopColumns`, `phoneColumns`, and `mobileColumns`?

Reply:
Ya you're right - I think phone and mobile are synonymous. Looks good.